css/screens/categories.css

```diff
@@ -0,0 +1,124 @@
+/* === css for categories page === */
+
+/* css for header */
+.big-wrapper{
+    padding: 1.7rem 0 2rem;
+    width: 100%;
+    min-height: 100vh;
+    overflow: hidden;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+}
+.container{
+    max-width: 81rem;
+    width: 100%;
+    margin: 0 auto;
+    padding: 0 3rem;
+}
+header .container{
+    position: relative;
+    z-index: 10;
```

Comment:
Any particular reason for keeping a high Z - Index? Any other elements conflicting? 

If not I suggest keeping it in the single digits.

Reply:
sure will remove that